examples/flutter_gallery/lib/demo/fortnightly/fortnightly.dart

```diff
@@ -0,0 +1,232 @@
+              child: Image.asset(
+                'food/fruits.png',
+                package: 'flutter_gallery_assets',
+                fit: BoxFit.fitWidth,
+              ),
+            ),
+            const SizedBox(height: 17),
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 8),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Row(
+                    children: <Widget>[
+                      Text(
+                        'US',
+                        style: textTheme.overline,
+                      ),
+                      Text(
+                        ' ¬ ',
+                        style: textTheme.overline.apply(
+                            color: Theme.of(context).textTheme.display3.color),
+                      ),
+                      Text(
+                        'CULTURE',
+                        style: textTheme.overline,
+                      ),
+                    ],
+                  ),
+                  const SizedBox(height: 10),
+                  Text(
+                    'Quince for Wisdom, Persimmon for Luck, Pomegranate for Love',
+                    style: textTheme.display1,
+                  ),
+                  const SizedBox(height: 10),
+                  Text(
+                    'How these crazy fruits sweetened our hearts, relationships,'
+                        'and puffed pastries',
+                    style: textTheme.body1,
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.symmetric(vertical: 16),
+                    child: Row(
+                      children: <Widget>[
+                        CircleAvatar(
+                          backgroundImage: ExactAssetImage(
+                            'people/square/trevor.png',
+                            package: 'flutter_gallery_assets',
+                          ),
+                          radius: 20,
+                        ),
+                        const SizedBox(width: 12),
+                        Text(
+                          'by',
+                          style: textTheme.display3,
+                        ),
+                        const SizedBox(width: 4),
+                        Text(
+                          'Connor Eghan',
+                          style: TextStyle(
+                            fontFamily: 'Merriweather',
+                            fontSize: 18,
+                            fontWeight: FontWeight.w500,
+                            color: Colors.black,
+                          ),
+                        )
+                      ],
+                    ),
+                  ),
+                  Text(
+                    'Have you ever held a quince? It\'s strange; covered in a fuzz '
+                        'somewhere between peach skin and a spider web. And it\'s '
```

Comment:
In the Flutter sources, we don't indent the second line of a long string like this (here and elsewhere).

Not also that you can create long strings like this, and you don't need to escape the apostrophes:
```
const String articleText = '''
Have you ever held a quince? It's strange; covered in a fuzz 
somewhere between peach skin and a spider web. And it's
''';
```

Reply:
Looks like when I do that, it takes my new lines literally:
![screen shot 2019-02-19 at 8 18 33 pm](https://user-images.githubusercontent.com/1271525/53059070-9bc12f80-3483-11e9-8f74-454e07824a9c.png)
![simulator screen shot - iphone xs max - 2019-02-19 at 20 18 39](https://user-images.githubusercontent.com/1271525/53059069-9bc12f80-3483-11e9-987f-32609cde8536.png)

Reply:
Good point. Using the triple-quote would be a little trickier. On the upside, there's no need for all of that escaping.

```
Text('$paragraph1\n\n$paragraph2')

final String paragraph1 = '''Have you ever held a quince?
 It's strange; covered in a fuzz somewhere between peach skin and a
 spider web. And it's hard as soft lumber. You'd be forgiven for
 thinking it's veneered Larch-wood. But inhale the aroma and you'll
 instantly know you have something wonderful. Its scent can fill a
 room for days. And all this before you've even cooked it.
'''.replaceAll('\n', '');


final String paragraph2 = '''Pomegranates on the other hand
 have become almost ubiquitous. You can find its juice in any bodega,
 Walmart, and even some gas stations. But at what cost? The
 pomegranate juice craze of the aughts made "megafarmers" Lynda and
 Stewart Resnick billions. Unfortunately, it takes a lot of water to
 make that much pomegranate juice. Water the Resnicks get from their
 majority stake in the Kern Water Bank. How did one family come to
 hold control over water meant for the whole central valley of
 California? The story will shock you.
'''.replaceAll('\n', '');
```

Reply:
I see what you did there. I'ma try that out.

Reply:
Done!